ethereum_history_api/oracles/src/script/generateVK.ts

```diff
@@ -1,8 +1,8 @@
 import { MonorepoCircuit } from '../noir/circuit/circuit.js';
 import { VerificationKey, generateVk } from '../noir/circuit/vk.js';
 
-const circuit = await MonorepoCircuit.create('../../', 'get_header');
-await generateVk(circuit);
-const vk = await VerificationKey.create(circuit);
+const circuit = await MonorepoCircuit.create('../../', 'get_storage');
```

Comment:
@LogvinovLeon Would this reference to monorepo circuit work when we release oracle server as separate NPM?

Would be cool if we can already use it easily from our js sdk.

Reply:
Those scripts use `nargo` and `barretenberg` which are both unavailable on the client, so I think that they will be useless in the library either way